.circleci/config.yml

```diff
@@ -48,6 +48,14 @@ jobs:
       - checkout
       - run: go get golang.org/x/lint/golint
       - run: make lint
+  fuzzit:
+    docker:
+      - image: fuzzitdev/golang:1.12.7-buster
```

Comment:
Would there be any value in using something like the latest tag instead of this?

Reply:
so the Dockerfile is [here](https://github.com/fuzzitdev/fuzzit/tree/master/docker/debian/stretch/golang12-llvm9)

the tag is the version of golang and it's always the latest (we push to that tag if there are changes) though it doesn't really change often as it just golang+clang docker

Reply:
I think it'd be nice to have it automatically update, could we make it latest?

Reply:
what do you mean by latest? latest golang or latest golang:1.12.7

Reply:
`fuzzitdev/golang:latest`. Is this not a tag you will support?